Fix:
error: implicitly declaring library function 'finite' with type 'int (double)' [-Werror,-Wimplicit-function-declaration]
--- t210/sopen_heka_read.c.orig	2017-07-23 15:40:06.000000000 -0500
+++ t210/sopen_heka_read.c	2022-01-24 12:41:34.000000000 -0600
@@ -22,6 +22,7 @@
 
 #include <assert.h>
 #include <ctype.h>
+#include <math.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/stat.h>
